build/packaging/WIX.template.in

```diff
@@ -136,6 +142,22 @@
                     <MIME Advertise="no" ContentType="application/x-musescore+xml" Default="no" />
                 </Extension>
             </ProgId>
+
+            <RegistryValue Root="$(var.RegistryRoot)" Key="SOFTWARE\Classes\MuseScore.mscz," Value="Compressed MuseScore Backup File" Type="string"/>
+            <ProgId Id="MuseScore.mscz," Description="Compressed Backup MuseScore File" Advertise="no" Icon="$(var.ExeId)" IconIndex="1">
+                <Extension Id="mscz," Advertise="no">
+                    <Verb Id="open" TargetFile="$(var.ExeId)" Command="Open" Argument="&quot;%1&quot;" />
+                    <MIME Advertise="no" ContentType="application/x-musescore" Default="yes" />
+                </Extension>
+            </ProgId>
+
+            <RegistryValue Root="$(var.RegistryRoot)" Key="SOFTWARE\Classes\MuseScore.mscx," Value="MuseScore Backup File" Type="string" />
+            <ProgId Id="MuseScore.mscx," Description="MuseScore Backup File" Advertise="no" Icon="$(var.ExeId)" IconIndex="1">
+                <Extension Id="mscx," Advertise="no">
+                    <Verb Id="open" TargetFile="$(var.ExeId)" Command="Open" Argument="&quot;%1&quot;" />
+                    <MIME Advertise="no" ContentType="application/x-musescore+xml" Default="no" />
+                </Extension>
+            </ProgId>
```

Comment:
The Windows part of the commit doesn't seem to work, WIX doesn't seems to like the trailing commas, see daa5fb7